lib/schemaUtils.js

```diff
@@ -1082,33 +1082,33 @@ module.exports = {
           helper = {
             type: 'basic',
             basic: [
-              { key: 'username', value: '<Basic Auth Username>' },
-              { key: 'password', value: '<Basic Auth Password>' }
+              { key: 'username', value: '{{basic_auth_username}}' },
```

Comment:
@erfemega It just occurred to me that we're using camel case in other environment variable such as `baseUrl`. Following that we should also keep camel casing here. i.e. `basicAuthUsername` etc. We can do same for all following cases.